test/endtoend/unmanaged_tablet_test.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+source ./tools/test.env
+source ./test/endtoend/utils.sh
+
+# get_started_unmanaged:
+function get_started_unmanaged() {
+    echo "Apply latest operator-latest.yaml"
+    kubectl apply -f "operator-latest.yaml"
+    checkPodStatusWithTimeout "vitess-operator(.*)1/1(.*)Running(.*)"
+
+    echo "Apply 101_initial_cluster_unmanaged_tablet.yaml"
+    kubectl apply -f "101_initial_cluster_unmanaged_tablet.yaml"
+    checkPodStatusWithTimeout "example-zone1-vtctld(.*)1/1(.*)Running(.*)"
+    checkPodStatusWithTimeout "example-zone1-vtgate(.*)1/1(.*)Running(.*)"
+    checkPodStatusWithTimeout "example-etcd(.*)1/1(.*)Running(.*)" 3
+    checkPodStatusWithTimeout "example-vttablet-zone1(.*)1/1(.*)Running(.*)"
+
+    sleep 10
+    echo "Creating vschema and commerce SQL schema"
+
+    ./pf_vtadmin.sh > /dev/null 2>&1 &
+    sleep 5
```

Comment:
Since we are not defining vtadmin in the yaml file, we should use the normal port forwarding instead of the one that includes vtadmin.

```suggestion
    ./pf.sh > /dev/null 2>&1 &
    sleep 5
```